doc/installation.rst

```diff
@@ -41,10 +41,10 @@ to complete the installation.
 Verifying the installation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-On all platforms, the command ``pyinstaller`` should now exist on the
+On all platforms, the command ``PyInstaller`` should now exist on the
 execution path. To verify this, enter the command::
 
-    pyinstaller --version
+    PyInstaller --version
 
 The result should resemble ``4.n`` for a released version,
 and ``4.n.dev0-xxxxxx`` for a development branch.
```

Comment:
Maybe we should add a note about using `python -m PyInstaller` when PyInstaller command cannot be found due to scripts directory not being in `PATH`?

Reply:
And actually, the shim is called `pyinstaller` on case-sensitive OSes, so the change is incorrect.

Reply:
The refactor from `pyinstaller` to `PyInstaller` was accidental. I've reverted it. And see other comments on this file.